fips/lib.sh

```diff
@@ -287,7 +225,7 @@ true <<'=cut'
 =head1 VARIABLES
 
 =head2 fipsMode
-
+u
```

Comment:
What does this `u` do?

Reply:
Ou oh, good chatch, I guess I was too hasty with merging this one.

Reply:
It is a harmless typo, fortunately (it is in pod "docstring").